### ISSUE TITLE
Flatten the too long questions

### ISSUE THREAD

**Issue by Pyscowicz** [Macronair/TheMillionaireGame#78]
I have a feature idea. Is it possible to make questions and answers flatten when they are too long to fit on the question and answer panels. In the old British version when one answer was too long it was flattened but also other answers were but in old American version when one answer was too long, only it was flattened and no others. I like more British style. Questions and answers font size is not meant to be changed. In the old Super Millionaire game that was made by C++ had that feature and I think it's not possible on VB but I hope It can be made.

**Comment by Grzegorz213769** [comment]
I agree! This is starting to bother me too. Although I disagree that flattening the text would be better. Given the lengthy replies, even that won't help. In my opinion, a better solution is to gradually shrink the text. Alternatively, an option could be introduced to control whether the text is reduced or flattened. That would be the best solution, as it would make everyone happy.
